native/packages/react-native-bpk-component-carousel/readme.md

```diff
@@ -0,0 +1,58 @@
+# react-native-bpk-component-carousel
+
+> Backpack React Native Carousel component.
+
+## Installation
+
+```sh
+npm install react-native-bpk-component-carousel --save-dev
+```
+
+## Usage
+
+```js
+import React, { Component } from 'react';
+import { View, StyleSheet } from 'react-native';
+import BpkCarousel, { BpkCarouselItem } from 'react-native-bpk-component-carousel';
+
+export default () => (
+  <BpkCarousel 
+    accessibilityLabel={(page, total) => `${page + 1} of ${total}`} 
+    style={{ width: 300, height: 300 }}
+    renderPagination={(page, total) => (
+      <Text>{page + 1} of {total}</Text>
+    )}
+  >
+    <BpkCarouselItem>
+      <Image
+        resizeMode="contain"
+        style={{ width: '100%', height: '100%' }}
+        source={{ uri: 'https://placekitten.com/g/300/300' }}
+      />
+    </BpkCarouselItem>
+    <BpkCarouselItem>
+      <Image
+        resizeMode="contain"
+        style={{ width: '100%', height: '100%' }}
+        source={{ uri: 'https://placekitten.com/300/300' }}
+      />
+    </BpkCarouselItem>
+  </BpkCarousel>
+);
+```
+
+## Props
+
+| Property           | PropType                              | Required | Default Value |
+| ------------------ | ------------------------------------- | -------- | ------------- |
+| accessibilityLabel | oneOfType(string, func)               | true     | -             |
+| children           | node                                  | true     | -             |
+| renderPagination   | func                                  | false    | null          |
```

Comment:
Is this the best API design? It seems like `BpkPaginationDots` should be more first party even if we support custom pagination indiactors too

Reply:
Agreed I think the pagination dots should be baked in with the ability to customise removed completely.

Reply:
Lol what a terrible example from me! Ah well..

Reply:
Your example in the stories are doing more than you say here. For example setting the style


```
 <BpkPaginationDots
    style={{
     alignSelf: 'center',
     marginTop: -spacingMd * 4,
   }}
```

This definitely feels like something a consumer shouldn't have to care about. At the very least if we make it configurable the `BpkPaginationDots` should render in the correct place by default

Reply:
Yeah what I meant with my example is that with the current implementation we don't need to care/impose this style. In any case I'll chat with @willhg tomorrow to check what are his views on this and what is the default style we want to have